views/topic/topic.hbs

```diff
@@ -69,6 +69,8 @@
                                         <a href="{{this.url}}" target="_blank">
                                             {{this.title}}
                                         </a>
+                                        <h5 style="color: #ff1134">{{$t "topic._topic.leavePage.warningMain" }}</h5>
```

Comment:
In the styles/courses folder there are some .scss files in which you can include these styles inside clases instead of placing them here. Also use the proper variable for this color **$colorDanger** which is defined in colors.scss